<documentation title="Cast Structure Spacing">
    <standard>
    <![CDATA[
    A type cast should be preceded by whitespace.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: space before typecast.">
        <![CDATA[
$a =<em> </em>(int) '420';
        ]]>
        </code>
        <code title="Invalid: no space before typecast.">
        <![CDATA[
$a <em>=(</em>int) '420';
        ]]>
        </code>
    </code_comparison>
</documentation>
